test/powershell/Modules/Microsoft.PowerShell.Utility/XMLCommand.Tests.ps1

```diff
@@ -49,4 +49,111 @@ Describe "XmlCommand DRT basic functionality Tests" -Tags "CI" {
 		$results.Property1 | Should Be $property1
 		$results.Property2 | Should Be $property2
     }
+
+	It "Export-Clixml StopProcessing should succeed" {
+		$ps = [PowerShell]::Create()
+		$ps.AddCommand("Get-Process")
+		$ps.AddCommand("Export-CliXml")
+		$ps.AddParameter("Path", $testfile)
+		$ps.BeginInvoke()
+		$ps.Stop()
+		$ps.InvocationStateInfo.State | Should Be "Stopped"
+	}
+
+	It "Import-Clixml StopProcessing should succeed" {
+		1,2,3 | Export-Clixml -Path $testfile
+		$ps = [PowerShell]::Create()
+		$ps.AddCommand("Get-Process")
+		$ps.AddCommand("Import-CliXml")
+		$ps.AddParameter("Path", $testfile)
+		$ps.BeginInvoke()
+		$ps.Stop()
+		$ps.InvocationStateInfo.State | Should Be "Stopped"
+	}
+
+	It "Export-Clixml using -Depth should work" {
+		class Three
+		{
+			[int] $num = 3;
+		}
+
+		class Two
+		{
+			[Three] $three = [Three]::New();
+		}
+
+		class One
+		{
+			[Two] $two = [Two]::New();
+		}
+
+		$one = [One]::New()
+		$one | Export-Clixml -Depth 2 -Path $testfile
+		$deserialized_one = Import-Clixml -Path $testfile
+		$deserialized_one.two.three.num | Should BeNullOrEmpty
```

Comment:
member access does not produce exception, unless strict mode is set. _Method_ invocation will throw null reference exception.